Assets/RGLUnityPlugin/Scripts/LidarSensor.cs

```diff
@@ -58,6 +58,9 @@ public class LidarSensor : MonoBehaviour
         [Tooltip("Allows to quickly enable/disable velocity distortion")]
         public bool applyVelocityDistortion = false;
 
+        [Tooltip("If disable, both beam divergence values are set to 0. Otherwise, they are set based on LiDAR configuration.")]
+        public bool simulateBeamDivergence = true;
```

Comment:
The default value should be decided.

Reply:
For the feature branch, I set this to false - this is exactly the approach that was used before. Final decision will be made before merge to main branch.